modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/index.d.ts

```diff
@@ -70,61 +69,27 @@ interface BaseItem {
 	status: string;
 }
 
-type ApplicationStatusKeys = 'published' | 'unpublished';
-
-interface ApplicationStatus {
-	key: ApplicationStatusKeys;
-	name?: 'Published' | 'Unpublished';
-}
-interface APIApplicationItem extends BaseItem {
-	applicationStatus: ApplicationStatus;
-	baseURL: string;
-	title: string;
-	version: string;
+interface FDSActionData {
```

Comment:
@lhine Sorry for not pointing it out before, but FDSActionData and FDSItem<T> should be declared right after `APIURLPaths`, because capitalized characters go before uncapitalized ones.